portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -52,7 +64,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the input from the comments form
     String authorText = getParameter(request, "author");
     String valueText = getParameter(request, "value");
-    comments.add(new Comment(authorText, valueText));
+    Comment comment = new Comment(authorText, valueText);
+
+    // store the comments in a datastore
+    Entity commentEntity = new Entity("Comment");
```

Comment:
Probably belongs in the database class

Reply:
I agree with Will on making a new class and placing the methods that I suggested within

Reply:
Created a new CommentDatabase class.